Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/PinchSlider.cs

```diff
@@ -0,0 +1,198 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.Events;
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// Slider with a thumb (a GameObject), endpoints are defined by two GameObjects
+    /// </summary>
+    public class PinchSlider : MonoBehaviour, IMixedRealityPointerHandler, IMixedRealityFocusHandler
+    {
+
+        #region Serialized Fields and Properties
+        [Tooltip("The gameObject that contains all slider children. Moves the sliders together.")]
+        [SerializeField]
+        private GameObject sliderRoot = null;
```

Comment:
I was a bit confused by this because I took _slider_ to be the whole thing not just the bit that moves. I think you are calling it thumb above, would it make more sense to call this member thumbRoot?

Reply:
Great point! I wil update.